tests/rptest/tests/describe_topics_test.py

```diff
@@ -313,6 +313,12 @@ def test_describe_topics_with_documentation_and_types(self):
                 value="dlq_table",
                 doc_string=
                 "Action to take when an invalid record is encountered."),
+            "redpanda.iceberg.target.lag.ms":
+            ConfigProperty(
+                config_type="LONG",
+                value="10",
+                doc_string="Something just tell me by failing",
```

Comment:
:D

Reply:
lol. and of course it didn't fail, so I forgot about it!

Reply:
:D, adding new properties is surprisingly complicated, so many places to update.

Reply:
oh, i think it's even a bit weirder than that. At least half of the `ConfigProperty`s here never appear in the describe response as formulated, so they are just dead code afaict. 

Looks to be a case of many properties not appearing in describe responses unless explicitly requested? I'll see whether I can fix it up async with this diff.